spec/unit/blacklight/access_controls/search_builder_spec.rb

```diff
@@ -8,6 +8,8 @@
   let(:user) { User.new }
   let(:ability) { Ability.new(user) }
 
+  before { allow(controller).to receive(:blacklight_config) }
+
```

Comment:
I don't think this is needed anymore.  In the github actions PR I implemented this in a different way based upon Blacklight's search builder spec tests.

Reply:
Removed.